client/hotrod-client/src/main/java/org/infinispan/client/hotrod/near/BoundedConcurrentMapNearCache.java

```diff
@@ -20,8 +22,8 @@ private BoundedConcurrentMapNearCache(ConcurrentMap<K, MetadataValue<V>> cache)
    }
 
    public static <K, V> NearCache<K, V> create(final NearCacheConfiguration config) {
-      return new BoundedConcurrentMapNearCache<K, V>(
-         CollectionFactory.<K, MetadataValue<V>>makeBoundedConcurrentMap(config.maxEntries()));
```

Comment:
Sadly this is probably one of the few methods I think that might be useful to keep around. I don't think everyone should have to know how to mess with Caffeine for just a simple bounded concurrent map.

Maybe just move to `Util`?

Reply:
Do we need a bounded concurrent map that often @wburns?

Reply:
I don't think it's worth adding a helper method for, as we only use it twice ourselves and I can't imagine many users require it. I think we should deprecate and then provide an alternative if it's ever requested.

Reply:
Okay, it seemed cleaner not having to have the client worry about caffeine though as well. But I am fine about not having it.